client/ayon_substancepainter/plugins/create/create_textures.py

```diff
@@ -152,6 +154,11 @@ def get_instance_attr_defs(self):
                     label="Review",
                     tooltip="Mark as reviewable",
                     default=True),
+            BoolDef("exportTextureSetsAsOneOutput",
```

Comment:
I feel the key here is very verbose, and also the label. Is "flatten texture set" clearer maybe?